Signal/src/call/CallService.swift

```diff
@@ -290,6 +291,10 @@ protocol CallServiceObserver: class {
         return getIceServers().then { iceServers -> Promise<HardenedRTCSessionDescription> in
             Logger.debug("\(self.TAG) got ice servers:\(iceServers)")
 
+            guard self.call == call else {
```

Comment:
The incoming call logic was already checking for obsolete calls, but the outgoing call logic wasn't.